### ISSUE TITLE
I have random/someone else's devices in my toshiba integration

### ISSUE THREAD

**Issue by MadMirza** [h4de5/home-assistant-toshiba_ac#200]
hi,

originally I only have 4 devices in my house and connected to my toshiba app and home assistant.
Out of nowhere, my devices in toshiba app is missing and I have 11 more devices connected to my toshiba integration in home assistant,  look like they are named in german or something, and mine is in english.

what is going on? any help is greatly appreciated.

![image](https://github.com/user-attachments/assets/db80c7a2-2aa1-4587-94a9-663edcdfbee7)
The first four are my devices.

HA Version: 2024.6
Toshiba integration version: was 2024.6 when issue occurred, updated to 2024.8 with no fix.

**Comment by h4de5** [comment]
the integration only lists whats returned from the cloud api. the list in the official toshiba app also changed?

**Comment by MadMirza** [comment]
@h4de5 yeah, all my devices from toshiba app is gone, also unsure why. No changes were made to it from my part :(

trying to get support from toshiba support, but sometimes they speak as if they have no knowledge of what they are saying.
